### ISSUE TITLE
DAISY / printdisabled workflow is confusing. Is it current?

### ISSUE THREAD

**Issue by hornc** [internetarchive/openlibrary#9439]
### Problem

Currently only some records display the option to download a DAISY file in the left side bar.

I can see them in works or editions with archive.org identifiers and which have the status "Not in library" or "Read".

#### Relevant URL(s)
Not in library example:
https://openlibrary.org/books/OL23174763M/Focus_and_background_in_Romance_languages

Read example:
https://openlibrary.org/works/OL1796093W/Fair_Margaret

#### Evidence / Screenshot
![image](https://github.com/internetarchive/openlibrary/assets/905545/ad1e1d4f-8500-432a-9a0c-a413252ca687)

Following through to the DAISY page e.g. https://openlibrary.org/books/OL24178528M/Fair_Margaret/daisy
the links to the DAISY files 404s.

There's a lot of code, translatable strings, and FAQs relating to this feature, 

* https://github.com/search?q=repo%3Ainternetarchive%2Fopenlibrary+daisy&type=code
*  https://github.com/internetarchive/openlibrary/blob/8b0f5ecdd1cd7e7088c19ac8e3e1d60eca1a95eb/openlibrary/i18n/messages.pot#L2674
* https://openlibrary.org/help/faq/accessing

### Reproducing the bug

* Expected behavior:

Following the instructions in the FAQ https://openlibrary.org/help/faq/accessing
leads to two subject pages:
https://openlibrary.org/subjects/accessible_book
https://openlibrary.org/subjects/protected_daisy

Which lists books to Borrow and to Read.

I'd expect
* The accessibility FAQ to be up to date and describe the current offering for accessible material, whether its DAISY or something else.
* Borrowable books to have a DAISY / print disability option. Currently they don't. e.g. https://openlibrary.org/works/OL29983W/Little_Women
* Preview only books to have a print disabled option. https://openlibrary.org/works/OL554620W/The_Duke_and_I
The above two examples are taken from the first six results on the 'protected_daisy' page.

* When I get to a /daisy page (by other means, because the FAQ and subject pages didn't help), the daisy.zip link should work: https://openlibrary.org/books/OL24178528M/Fair_Margaret/daisy  It doesn't.

* Actual behavior:
Stuff doesn't work as advertised. It's not clear if the instructions are out of date, or some things are not working as they should. It's not clear how a patron should access print-disabled material, or what one should expect. 

As far as I know the links to https://www.loc.gov/nls/how-to-enroll/apply-for-nls-services/#eligibility-guidelines (which still resolve) all relate to current plans and these programs are still running.

Archive.org still offers protect DAISY files, for that item:
https://archive.org/details/cu31924021992015
![image](https://github.com/internetarchive/openlibrary/assets/905545/d6b4b113-fc1a-49d6-822a-9aa8a138a1f8)

but there isn't a pre-generated one at the URL offered by the OL daisy page.

Also, I now notice the main search page has a **print-disabled** option. In the context of this issue I don't see what the filtering does, or how the results provide print-disabled access (any more or less than any other search result item) https://openlibrary.org/search?q=happiness&mode=printdisabled&subject_facet=Protected+DAISY

![image](https://github.com/internetarchive/openlibrary/assets/905545/d67b32ba-c15f-47da-b650-11fb2ec533ec)

### Context

- Logged in (Y/N): Y
- Environment (prod, dev, local): prod

#### Stakeholders

**Note:** Before [making a new branch](https://github.com/internetarchive/openlibrary/wiki/Git-Cheat-Sheet#making-changes-and-creating-a-pull-request) _or_ updating an existing one, please [ensure your branch is up to date](https://github.com/internetarchive/openlibrary/wiki/Git-Cheat-Sheet#working-on-your-branch).

**Comment by mekarpeles** [comment]
Related also to #5831

**Comment by mekarpeles** [comment]
Agreed this flow is broken, @bfalling may wish to weigh in re: how we support daisy. Suggesting for now we update the OL daisy link to redirect to the ia item instead of the daisy redeem page..

**Comment by vshOps** [comment]
Hi! I'd like to work on this issue if it's still available. I'm new to contributing to Open Library and would be happy to investigate and submit a fix.

**Comment by mekarpeles** [comment]
Now redirects to the archive.org page; Leaving it there for now
